report-hazard/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <script src="report-hazard.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
+	<link href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- <script type="module" src="script.js"></script> -->
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>CampBuddy</h1>
+    </header>
+    <div class="toolbar">
+        <!-- header area shared among all pages. It also include navigation buttons -->
+        <a title="Select the location" href="#step1">1</a>
+        <span>-----</span>
```

Comment:
remove the spans, this will be done with css after

also this should be a nav with ul li and inside the a tags

Reply:
commit a57619a71caed49e2cafb2e1afed2239098bd054